tests/test_content_guard_rbac.yml

```diff
@@ -0,0 +1,163 @@
+---
+- name: Test content guard rbac
+  gather_facts: false
+  hosts: localhost
+  vars:
+    pulp_url: http://localhost:8080
+    pulp_username: admin
+    pulp_password: password
+    pulp_validate_certs: true
+  tasks:
+
+    - name: Query pulp status
+      uri:
+        url: "{{ pulp_url }}/pulp/api/v3/status/"
+        user: "{{ pulp_username }}"
+        password: "{{ pulp_password }}"
+        method: GET
+        body_format: json
+        status_code: 200
+        force_basic_auth: true
+      register: status_result
+
+    - name: Exit if version < 3.17
```

Comment:
Current tests fail because the entire point of this change is to update the collection to take advantage of the updated API. Is it better to remove the github action or skip the test with this?

Reply:
You can skip the test in this case